### PR TITLE
Bug 1876363 - Ship v125.0.0 of the WebCompat System Addon

### PR DESCRIPTION
**Pull request by ksy36**
https://bugzilla.mozilla.org/show_bug.cgi?id=1876363

### CODE REVIEW

File: src/data/injections.js [hunk 1]
```diff
@@ -540,19 +508,19 @@ const AVAILABLE_INJECTIONS = [
         "*://*.avanabayview.com/*", // #118617
         "*://*.breakpointeandcoronado.com/*", // #117735
         "*://*.courtsatspringmill.com/*", // #128404
+        "*://*.fieldstoneamherst.com/*", // #131703
```
Comment (wisniewskit) [inline src/data/injections.js:511]: I think this is supposed to be 132974, yes?

**Review by wisniewskit — APPROVED** [review]
Just one minor note about a mismatched bug number, otherwise good to go!

File: src/data/injections.js (hunk 1 above)
Comment (ksy36) [reply src/data/injections.js]: good catch, thanks!

**Pull request merged**